ui/test/nuxeo-favorites-toggle-button.test.js

```diff
@@ -84,13 +92,29 @@ suite('nuxeo-favorites-toggle-button', () => {
         '{"entity-type": "document","uid": "1"}',
       ]);
     });
+    teardown(() => {
+      if (window.confirm.restore) {
+        window.confirm.restore();
+      }
+    });
+
+    test('toggle should not remove document from favorites when clicked Cancel', async () => {
+      sinon.stub(window, 'confirm').returns(false);
+      expect(element.favorite).to.be.true;
+    });
```

Comment:
This test assumes that the document is in favorites. Therefore, it should be in the other test suite "when a document is a favorite"